setup.py

```diff
@@ -0,0 +1,9 @@
+"""setup file"""
+from distutils.core import setup
+
+setup(
+    name='sem_size_analysis',
+    version='0.1',
```

Comment:
It's a good idea to have your version in your config.py file so you can toggle it like the other configurable params.